doc/vagrant/share/packelektra.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
```

Comment:
I think I will remove this file anyway. This file was meant as a showcase for how one can use containers/VMs. I think its better if I incorporate and describe the functionality of this file into the README.